src/Components/v2/CarouselV2.tsx

```diff
@@ -0,0 +1,265 @@
+import { Box, ChevronIcon, Flex } from "@artsy/palette"
+import React, { ReactNode, useEffect, useRef, useState } from "react"
+import styled from "styled-components"
+import { left, LeftProps, right, RightProps } from "styled-system"
+import { Media } from "Utils/Responsive"
+
+export interface FlickitySettings {
```

Comment:
<img width="439" alt="Screen Shot 2019-04-26 at 10 23 45 AM" src="https://user-images.githubusercontent.com/236943/56825213-659a8600-680d-11e9-8ed2-6b4e42d6d6f2.png">

These types are available through http://definitelytyped.org/, which has Types for most JS libs.

`yarn add -D @types/flickity`

Reply:
Added!